packages/kbn-analytics/src/metrics/index.ts

```diff
@@ -19,15 +19,18 @@
 
 import { UiStatsMetric } from './ui_stats';
 import { UserAgentMetric } from './user_agent';
+import { ApplicationUsageCurrent } from './application_usage';
 
 export { UiStatsMetric, createUiStatsMetric, UiStatsMetricType } from './ui_stats';
 export { Stats } from './stats';
 export { trackUsageAgent } from './user_agent';
+export { ApplicationUsage, ApplicationUsageCurrent } from './application_usage';
 
-export type Metric = UiStatsMetric | UserAgentMetric;
+export type Metric = UiStatsMetric | UserAgentMetric | ApplicationUsageCurrent;
```

Comment:
Aside and not for this PR: We might want to consider adding a generic type that devs can use for metrics not falling into any of the current metric types. For example: the recent File Upload migration to the NP includes a a request to add `File Upload` as a type. We should think about how to handle cases like these where we might not necessarily allow an overwrite of the generic type but rather have that if a generic type is used, there must be another descriptor (like a `description`) tag in the data we're capturing. e.g. metric type='generic', and description='File Upload`.

Reply:
That's a fair comment. I think the main reason why we haven't used the `UIStatsMetrics` more is because of the way we report them (as arrays). 

I'm sure we'll be coming back to this at some point with the new Pulse approach where arrays won't be an issue anymore.